git-make-release

```diff
@@ -98,28 +75,38 @@ elif [ "${LAST_VERSION_ARRAY[1]}" != "${MINOR}" ]; then
     echo "*** Warning: Minor release, but patch is not zero!"
   fi
 fi
+if [[ ${CURRENT_BRANCH} != master ]]; then
+  RELEASE_TYPE="BACKPORT"
+fi
 
 echo "Release type: ${RELEASE_TYPE}"
 
-# ask whether to proceed
-echo ""
-echo -n "Do you want to proceed with creating the tag (y/N)? "
-read ANSWER
-if [ "$ANSWER" != "y" ]; then
-  echo "Aborted."
-  rm -f $CHANGES
-  exit 1
-fi
+# obtain the change log since the latest tag
+CHANGES=`mktemp`
+git log ${LAST_VERSION}..HEAD --pretty=%s | grep -v '^project-template:' | grep -v '(project-template): '           \
+  | grep -v '^Merge pull request ' | grep -v '^Merge branch ' | grep -v '^Merge project-template (automated)'       \
+  | awk '{print "- " $0}' > $CHANGES
 
 # prepend header to the changelog
 CHANGES_WITH_HEADER=`mktemp`
 echo "New ${RELEASE_TYPE} release ${NEXT_VERSION}" > $CHANGES_WITH_HEADER
 echo "" >> $CHANGES_WITH_HEADER
 echo "Changes since ${LAST_VERSION}:" >> $CHANGES_WITH_HEADER
+echo "" >> $CHANGES_WITH_HEADER
 cat $CHANGES >> $CHANGES_WITH_HEADER
 
 # edit the changelog
-${EDITOR} ${CHANGES_WITH_HEADER}
+nano ${CHANGES_WITH_HEADER}
```

Comment:
Why do you force nano? What speaks against the programmer's default editor?

Reply:
sorry, that's a mistake caused by my file not directly coming from the repo... :-/

Reply:
There also was a line in the top defining the $EDITOR variable

Reply:
Also other things were missing, my version was just outdated, sorry. All this should be fixed now (I had to force-push though).